marketplace_builder/views/pages/how-platformos-works/basic-concepts.liquid

```diff
@@ -15,7 +15,7 @@ searchable: true
 
 **Partials** (partial templates) allow you to easily organize and reuse your code by extracting pieces of code to their own files. They help you improve code readability and follow the principle of DRY (Don’t Repeat Yourself). You can parameterize partials and use them in various places, e.g. layouts, pages, Authorization Policies, Form Configurations.
 
-[Learn how to use pages, layouts and partials in our Get Started guides](/get-started/pages)
+[Learn how to use pages, layouts and partials in our Get Started guides](get-started/pages/pages)
```

Comment:
I think leading slash wasnt a mistake

Reply:
@pavelloz looking at other links, you have it in some pages (e.g. related topics at the bottom of this page - https://documentation.platform-os.com/get-started/pages/pages).  But I have done it in some links and not others in this commit.   Let me know what you'd/ @diana-lakatos  would like and I'll update 😄

Reply:
Thank you for fixing these links! The leading slash is needed when we add links, we only omit them for slugs.